### ISSUE TITLE
Keys exposed to public

### ISSUE THREAD

**Issue by mathyoudawson** [d33pthought/node-totp#1]
Hello!

In `totp.js ` there are many secret keys that are exposed to the public.
I understand that this repo is old and these keys may have been rotated or removed, but just in case I think you should make this repo private.

Cheers :)

**Comment by d33pthought** [comment]
Yeah, I think they're old, but I'll delete the repo now just in case. Thanks.
